cypress/e2e/registration.cy.js

```diff
@@ -0,0 +1,44 @@
+/// <reference types='cypress' />
+
+describe('Cypress Homework 4', () => {
+  const user = 'UserMO';
+
+  beforeEach(() => {
+    cy.login();
+  });
+
+  it('Should contain username on the page.', () => {
+    cy.get('#userName-value').should('contain', user);
+  });
+
+  it('User can add and delete book form books users list', () => {
+    cy.get('#gotoStore')
+      .click();
+
+    cy.get('#searchBox')
+      .type('you');
+
+    cy.get('[id="see-book-You Don\'t Know JS"]')
+      .click();
+
```

Comment:
Add more assertions to the tests, right now there is no one `should`. You can check the title and author of the book, and that you delete a book with that particular name and author, and so on